cmd/advertisement-broadcaster/main.go

```diff
@@ -17,12 +19,18 @@ func main() {
 	flag.StringVar(&clusterId, "cluster-id", "", "The cluster ID of your cluster")
 	flag.StringVar(&gatewayIP, "gateway-ip", "", "The IP address of the gateway node")
 	flag.StringVar(&gatewayPrivateIP, "gateway-private-ip", "", "The private IP address of the gateway node")
-	flag.StringVar(&peeringRequestName, "peering-request", "", "Name of PeeringRequest CR where get configurations")
+	flag.StringVar(&peeringRequestName, "peering-request", "", "Name of PeeringRequest CR containing configurations")
 	flag.Parse()
 
+	log := ctrl.Log.WithName("setup")
 	ctrl.SetLogger(zap.New(func(o *zap.Options) {
 		o.Development = true
 	}))
 
+	if peeringRequestName == "" {
```

Comment:
Can we impose the --peering-request as mandatory instead of adding this control?

Reply:
According to [this](https://stackoverflow.com/questions/31786215/can-command-line-flags-in-go-be-set-to-mandatory#:~:text=The%20flag%20package%20does%20not,halt%20with%20an%20error%20message) it's not possible